src/include/nccl_common.h

```diff
@@ -17,7 +17,7 @@ typedef enum { ncclFuncBroadcast, ncclFuncReduce, ncclFuncAllGather, ncclFuncRed
 
 #define FUNC_INDEX_P2P 835
 #define FUNC_INDEX_ALLTOALL_PIVOT 555
-#define FUNC_INDEX_TOTAL 846
+#define FUNC_INDEX_TOTAL 992
```

Comment:
During build, rccl will generate host_table.cc in hipify/src/device which helps mapping func ids to appropriate indexes. In this case, it should be FUNC_INDEX_TOTAL=992, FUNC_INDEX_P2P=979, FUNC_INDEX_ALL_TO_ALL_PIVOT=651

Reply:
update FUNC_INDEX_P2P=979, FUNC_INDEX_ALL_TO_ALL_PIVOT=651 according to cr comments